vm/hugepage/libhugetlbfs/dependency.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# For glibc-static dependence issue on rhel8
+# Bug 1567712 - glibc-static needs to be multilib on RHEL 8
+
+this_arch=$(uname -m)
+this_distro=$(grep DISTRO /etc/motd | cut -d '-' -f 3)
+pre_distro=${this_distro%.*}
+end_distro=${this_distro##*.}
+end_distro=${end_distro// /}
+
+if grep -q "release 8.*" /etc/redhat-release; then
+        if echo $this_distro | grep -q '.n'; then
+		choose=nightly
+        else
+		choose=rel-eng
+        fi
+        min_rel=$(grep -oE "[0-9]\.[0-9]" /etc/redhat-release | cut -d. -f2)
+
+	for ((; end_distro >= 0; end_distro-- )) do
+		build_root=http://download.eng.bos.redhat.com/$choose/rhel-8/RHEL-8/RHEL-8.${min_rel}.0-${pre_distro}.${end_distro}/compose/CRB/$this_arch/os/
+		echo "Trying option one: $build_root"
+		if curl --fail --head -s $build_root >/dev/null; then
+			success=1
+			break
+		fi
+	done
+
+	((success)) || build_root=http://download.eng.bos.redhat.com/rel-eng/rhel-8/RHEL-8/latest-RHEL-8.${min_rel}.0/compose/CRB/$this_arch/os/
+
+	echo Using repo $build_root
+
+        cat << EOF > /etc/yum.repos.d/${repo_name:-buildroot.repo}
+[beaker-BUILDROOT]
+name=beaker-BUILDROOT
+baseurl=$build_root
+enabled=1
+gpgcheck=0
+skip_if_unavailable=1
+EOF
+
+	if [ $this_arch == x86_64 ]; then
+		rpm --quiet -q glibc-static || yum install -y glibc-static.x86_64 glibc-static.i686
+	else
+		rpm --quiet -q glibc-static || yum install -y glibc-static
+	fi
+
+	rm -fr /etc/yum.repos.d/buildroot.repo
```

Comment:
Hm, I'm curious about this: do we intend to delete the file we just created, or specifically `buildroot.repo`? Because the file created could have a different name, if `$repo_name` has a value...

And in any case, the `-r` is not needed, because it's always a file and not a directory?

Reply:
Hi @emanchado, file `dependency.sh` is just copied from [cgit/tests/kernel](http://pkgs.devel.redhat.com/cgit/tests/kernel/tree/vm/hugepage/libhugetlbfs/dependency.sh). And the Makefile looks pretty complicated, I'll re-write it, please wait for a while ...